jujugui/static/gui/src/app/utils/test-statsd.js

```diff
@@ -71,11 +71,24 @@ describe('StatsClient', () => {
     assertXHRSent(url, 'gui.awesome:1|c');
   });
 
-  it('can add flags as statsd tags', () => {
+  it('adds flags as statsd tags', () => {
     const url = 'https://example.com/stats/';
     const flags = {'test.bar': true, 'baz': true};
     const client = new window.jujugui.StatsClient(url, 'gui', flags);
     const name = client._addFlags('foo');
     assert.equal(name, 'foo,test.bar=true');
   });
+
+  it('adds multiple flags as statsd tags', () => {
+    const url = 'https://example.com/stats/';
+    const flags = {
+      'test.bar': true,
+      baz: true,
+      'test.bad': false,
```

Comment:
We never actually pass along inactive flags; these are always going to be true. We only put a value in so that we can create flags as an object when we're setting up the gui config on the blues browser side.